addons/common/functions/fnc_getMGRSdata.sqf

```diff
@@ -22,27 +22,38 @@ private _long = getNumber (configFile >> "CfgWorlds" >> _map >> "longitude");
 private _lat =  getNumber (configFile >> "CfgWorlds" >> _map >> "latitude");
 private _altitude =  getNumber (configFile >> "CfgWorlds" >> _map >> "elevationOffset");
 
-if (_map in ["Chernarus", "Bootcamp_ACR", "Woodland_ACR", "utes"]) then { _lat = 50; _altitude = 0; };
+if (_map in ["chernarus", "Chernarus_Summer", "chernarus_winter", "Bootcamp_ACR", "Woodland_ACR", "utes"]) then { _lat = 50; _altitude = 0; };
 if (_map in ["Altis", "Stratis"]) then { _lat = 40; _altitude = 0; };
-if (_map in ["Takistan", "Zargabad", "Mountains_ACR"]) then { _lat = 35; _altitude = 2000; };
+if (_map in ["takistan", "zargabad", "Mountains_ACR"]) then { _lat = 35; _altitude = 2000; };
 if (_map in ["Shapur_BAF", "ProvingGrounds_PMC"]) then { _lat = 35; _altitude = 100; };
 if (_map in ["fallujah"]) then { _lat = 33; _altitude = 0; };
-if (_map in ["fata", "Abbottabad"]) then { _lat = 30; _altitude = 1000; };
+if (_map in ["fata"]) then { _lat = 33; _altitude = 1347; };
+if (_map in ["Abbottabad"]) then { _lat = 34; _altitude = 0; };
```

Comment:
ACE already has something similar:
```sqf
     GVAR(mapAltitude)
     GVAR(mapLatitude)
     GVAR(mapLongitude)
```

the problem is just that the gps is not adding `GVAR(mapAltitude)` as it should (the DAGR, microDAGR and Kestrel are doing it properly already!)